main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -75,29 +77,37 @@ public static void main(String[] argv) {
         gtfsInput = GtfsInput.createFromPath(Paths.get(args.getInput()));
       }
     } catch (IOException | URISyntaxException | InterruptedException e) {
-      // FIXME: Add system errors to noticeContainer here.
+      noticeContainer.addSystemError(
+              new GtfsInputCreationError(e.getClass().getCanonicalName(), e.getMessage()));
+      generateAndExportReports(args, noticeContainer);
       logger.atSevere().withCause(e).log("Cannot load GTFS feed");
       return;
     }
     feedContainer =
-        feedLoader.loadAndValidate(gtfsInput, feedName, validatorLoader, noticeContainer);
+            feedLoader.loadAndValidate(gtfsInput, feedName, validatorLoader, noticeContainer);
+
+    // Output
+    generateAndExportReports(args, noticeContainer);
+    final long endNanos = System.nanoTime();
+    System.out.printf("Validation took %.3f seconds%n", (endNanos - startNanos) / 1e9);
+    System.out.println(feedContainer.tableTotals());
+  }
 
-    // Output.
+  /**
+   * Generates and exports reports for both validation notices and system errors reports
+   */
+  private static void generateAndExportReports(
+          final Arguments args, final NoticeContainer noticeContainer) {
```

Comment:
If output base is the only variable of Arguments used, maybe just pass that instead of the entire Arguments object instead?

Reply:
Indeed! Thanks.